templates/base.html

```diff
@@ -46,24 +41,10 @@
                   });
               });
           </script>
-          <!-- <script type="text/javascript">
-            $(document).ready(function () {
-                $("ul.navbar-nav > li > a").click(
-                  function (e) {
-                    $("ul.navbar-nav > li").removeClass(
-                      "active");
-                    $("ul.navbar-nav > li > a").css(
-                      "color", "");
- 
-                    $(this).addClass("active");
-                    $(this).css("color", "blue");
-                });
-            });
-        </script> -->
-        {% block content %}
-        {% endblock %}
         <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
         <script src="https://cdn.jsdelivr.net/npm/popper.js@1.14.7/dist/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
         <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+        {% block content %}
+        {% endblock %}
```

Comment:
removed and reordered unnecessary code

Reply:
The scripts and css for the navbar were there commented out to come back to later since the colour of the active page doesn't change in the navbar (captured in bug #42). It's cool to remove them, but we can also keep them in if you think it's a good reference to come back to later.

Reply:
yeah that will need to be addressed later, I have different ideas around that now.